Product.Service/Services/ItemObjectsService.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Product.Data;
+using Product.Schema.Models;
+using Product.Service.IServices;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Product.Service.Services
+{
+    public class ItemObjectsService : IItemObjectsService
+    {
+        private readonly ApplicationDbContext ProductDbContext;
+        Data.ItemObjects ItemObject;
+        public ItemObjectsService(ApplicationDbContext _dbContext)
+        {
+            ProductDbContext = _dbContext;
+            if (ProductDbContext != null)
+            {
+                ItemObject = new Data.ItemObjects(ProductDbContext);
```

Comment:
this is wrong I think
You should just sstore dbContext

Reply:
In services you shouldn't have dbContext.
Here you shuod have object of repositry

Reply:
done